app/graphql/collection_loader.rb

```diff
@@ -2,7 +2,7 @@ class CollectionLoader < GraphQL::Batch::Loader
   def initialize(model, association_name, scope = nil)
     @model = model
     @association_name = association_name
-    @scope = scope
+    @scope = scope || authorized_scope
```

Comment:
This means if a consumer of this class passes a scope it could bypass the authorization by mistake, e.g. in Resolvers::Domain::Subnets.

Reply:
Ok, this was a bad shortcut I took. A lambda that just extends the `authorized_scope` is a much better idea.